sympy/vector/deloperator.py

```diff
@@ -48,36 +35,27 @@ def gradient(self, scalar_field, doit=False):
 
         >>> from sympy.vector import CoordSysCartesian
         >>> C = CoordSysCartesian('C')
-        >>> C.delop.gradient(9)
-        (Derivative(9, C.x))*C.i + (Derivative(9, C.y))*C.j +
-            (Derivative(9, C.z))*C.k
+        >>> C.delop.gradient(C.x)
+        (Derivative(C.x, C.x))*C.i + (Derivative(C.x, C.y))*C.j +
+            (Derivative(C.x, C.z))*C.k
         >>> C.delop(C.x*C.y*C.z).doit()
         C.y*C.z*C.i + C.x*C.z*C.j + C.x*C.y*C.k
 
         """
-
-        scalar_field = express(scalar_field, self.system,
-                               variables=True)
-        vx = Derivative(scalar_field, self._x)
-        vy = Derivative(scalar_field, self._y)
-        vz = Derivative(scalar_field, self._z)
-
-        if doit:
-            return (vx * self._i + vy * self._j + vz * self._k).doit()
-        return vx * self._i + vy * self._j + vz * self._k
+        return gradient(scalar_field, doit=doit)
 
     __call__ = gradient
     __call__.__doc__ = gradient.__doc__
 
-    def dot(self, vect, doit=False):
+    def dot(self, vector, doit=False):
```

Comment:
I will revert this changes. I've read the document about deprecation policy, but I didn't suspect, that so small changes (`vect` -> `vector`) could be confusing. 
Definitely, this PR is not the place for that type of changes.